pkg/provider/aws/plugin.go

```diff
@@ -69,10 +69,18 @@ func (a *AWS) CopyConstructEdgesToDag(result *core.ConstructGraph, dag *core.Res
 }
 
 // configureResources calls every resource's Configure method, for resources that exist in the graph
-func (a *AWS) configureResources(dag *core.ResourceGraph) (err error) {
+func (a *AWS) configureResources(result *core.ConstructGraph, dag *core.ResourceGraph) (err error) {
 	var merr multierr.Error
 	for _, resource := range dag.ListResources() {
-		merr.Append(dag.CallConfigure(resource, nil))
+		var configuration any
+		if lambda, ok := resource.(*resources.LambdaFunction); ok {
```

Comment:
Just looking forward, should this be a type switch?

Reply:
yeah it likely should. This will eventually be done very different because we dont want to think about constructs but for now let me change this